components/encumbered/gst-editing-services1/gst-editing-services1.p5m

```diff
@@ -100,7 +100,6 @@ file path=usr/lib/$(MACH64)/libges-1.0.so.0.2201.0 \
 file path=usr/lib/$(MACH64)/pkgconfig/gst-editing-services-1.0.pc
 file path=usr/lib/python3.9/site-packages/gi/overrides/GES.py \
     pkg.tmp.autopyc=false
-file path=usr/share/bash-completion/completions/ges-launch-1.0
 file path=usr/share/gir-1.0/GES-1.0.gir
 file path=usr/share/gstreamer-1.0/validate/scenarios/ges-edit-clip-while-paused.scenario
 file path=usr/share/man/man1/ges-launch-1.0.1
```

Comment:
Without changing the Makefile our jenkins won't be triggered. Plus, you want to change the package contents and thus you'll need to change the COMPONENT_REVISION.